### PR TITLE
Update towncrier template

### PR DESCRIPTION
**Pull request by asvetlov**
Get rid of custom `tools/fix_changelog.py` postprocess procedure

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,5 +1,11 @@
+[build-system]
+requires = ["setuptools", "wheel"]
```
Comment (webknjaz) [inline pyproject.toml]: @asvetlov you were against it earlier, suggesting that sdist fallback should work, have you changed your mind?

Comment (asvetlov) [reply pyproject.toml]: I've copied the lines from `attrs` project honestly.
`sdist` fallback should still work.
We do `from setuptools import setup` already in `setup.py`.
[PEP  518](https://www.python.org/dev/peps/pep-0518/) recommends this lines.
Did I miss something?

Comment (webknjaz) [reply pyproject.toml]: Back when I proposed it you told me that fallback to pure python version won't work if the user does not have gcc. OTOH it was probably about cython presence there.

Comment (webknjaz) [reply pyproject.toml]: Anyway, I'd set s minimum setuptools version at least anyway.

Comment (asvetlov) [reply pyproject.toml]: I don't remember that discussion, sorry. 
Maybe it was about `setup.py`, not `pyproject.toml`?

What setuptools version should be used?

Comment (asvetlov) [reply pyproject.toml]: I've decided to drop the section: I want to backport the PR into 3.5.
Even a tiny chance of breaking build system doesn't worth it.

Comment (webknjaz) [reply pyproject.toml]: It wouldn't break the build. It's just about pip making a virtualenv and installing build deps there to use them in the next step. Without cython it wouldn't affect the build.

**Comment by asvetlov** [comment]
I recall the idea to add `cython` to build-system requirements list but the change is completely unrelated.
The idea is adding explicit lines for parameters already used implicitly.
But anyway it is the subject for another patch (if any).

**Pull request merged**